recipes-kernel/linux/5.10/patches/xsa-155-qsb-023-xen-netfront-copy-response-out-of-shared-buffer-befo.patch

```diff
@@ -18,9 +18,11 @@ use issue.
 ################################################################################
 PATCHES 
 ################################################################################
---- a/drivers/net/xen-netfront.c
-+++ b/drivers/net/xen-netfront.c
-@@ -387,13 +387,13 @@ static void xennet_tx_buf_gc(struct netf
+Index: linux-5.10.49/drivers/net/xen-netfront.c
+===================================================================
+--- linux-5.10.49.orig/drivers/net/xen-netfront.c
++++ linux-5.10.49/drivers/net/xen-netfront.c
```

Comment:
Please refresh this patch with a config matching the one found here: https://openxt.atlassian.net/wiki/spaces/DC/pages/20905986/Dealing+with+quilt

That cuts down on the version specific churn when refreshing patches.

Reply:
> Please refresh this patch with a config matching the one found here

Will do.

> Good find on strace. Is it a build time or run time incompatibility?

Likely both, build time in this case. strace seems to align itself with the same versioning as linux does (eg v5.10 strace is the first strace version to be compatible with v5.10 linux, and so on). v5.12 was the latest version at the time of our downstream work, so we went with that.

Reply:
Yes, adding the .quiltrc to the bordel docker image is a good idea.